frontend/src/components/inputs/EntityForm.vue

```diff
@@ -5,21 +5,20 @@
                  :required="true"/>
       <TextInput label="Slug" v-model="editEntity.slug" :args="{ id: 'slug', maxlength: 128 }"
                  :required="true"/>
-
       <h3 class="mt-3">Attributes</h3>
       <template v-for="attr in schema?.attributes || []" :key="attr.name">
         <!-- ATTRIBUTE IS REFERENCE -->
         <template v-if="attr.type === 'FK'">
-          <ReferencedEntitySelect v-model="editEntity[attr.name]" :label="attr.name"
-                                  :args="{id: attr.name}" :fk-schema-id="attr.bound_schema_id"
-                                  :select-type="attr.list ? 'many': 'single'"
-                                  :required="requiredAttrs.includes(attr.name)"/>
+            <ReferencedEntitySelect v-model="editEntity[attr.name]" :label="attr.name"
+                                    :args="{id: attr.name, tooltip: attr.description}" :fk-schema-id="attr.bound_schema_id"
+                                    :select-type="attr.list ? 'many': 'single'"
+                                    :required="requiredAttrs.includes(attr.name)" />
```

Comment:
Just a little cosmetics, if you don't mind.

```suggestion
          <ReferencedEntitySelect v-model="editEntity[attr.name]" :label="attr.name"
                                  :args="{id: attr.name, tooltip: attr.description}" :fk-schema-id="attr.bound_schema_id"
                                  :select-type="attr.list ? 'many': 'single'"
                                  :required="requiredAttrs.includes(attr.name)" />
```